src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ConvertToString.liquid

```diff
@@ -1,9 +1,15 @@
 ﻿{%              if GenerateNullableReferenceTypes -%}
+[return: System.Diagnostics.CodeAnalysis.NotNullIfNotNull("value")]
 private string? ConvertToString(object? value, System.Globalization.CultureInfo cultureInfo)
 {%              else -%}
 private string ConvertToString(object value, System.Globalization.CultureInfo cultureInfo)
 {%              endif -%}
 {
+    if (value is null)
```

Comment:
I try to use C# 6 and below (I think) isnt this a more recent feature?

Reply:
According to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/is#constant-pattern , it is C#7, will fix.